demos/tests/cases.py

```diff
@@ -353,9 +353,11 @@ def single_option_cases(key, *args):
         TestCase(options={'-no_show': None,
             **MONITORS,
             '-i': DataPatternArg('person-detection-retail')}),
+        *combine_cases(
+                TestCase(options={'-at': 'ssd'}),
         single_option_cases('-m_det',
             ModelArg('person-detection-retail-0002'),
-            ModelArg('person-detection-retail-0013')),
+            ModelArg('person-detection-retail-0013'))),
```

Comment:
it seems models.lst and test cases should be extended to cover new supported models